### PR TITLE
Bug Fixes

### PR DESCRIPTION
**Pull request by cs-sujeet-ahirrao**
0880731 - GitHub - List Repository Issue
0880733 - Set GitHub Username
0880711 - Pull Latest Content from GitHub
0880706 - Push Change to Source Control

### CODE REVIEW

File: playbooks/02 - Use Case - CICD/Pull Latest Content from GitHub.json [hunk 1]
```diff
@@ -802,6 +802,15 @@
...
+            "name": "Start -> Check Status",
+            "targetStep": "\/api\/3\/workflow_steps\/40f2c261-ea90-43e7-b7be-9e9a00685c7d",
+            "sourceStep": "\/api\/3\/workflow_steps\/9c013279-76d9-41bf-a80d-be525c0d9a43",
+            "label": null,
+            "isExecuted": false,
+            "uuid": "2e71c1db-0ac8-4ce0-a677-aca8bb3ef815"
+        },
         {
             "@type": "WorkflowRoute",
             "name": "Copy of Get Import Job -> Get Import Job",
```
Comment (cs-suyog-jinturkar) [inline playbooks/02 - Use Case - CICD/Pull Latest Content from GitHub.json:816]: Copy of Get Import Job -> Get Import Job --> you might want to change the step name in json.

File: playbooks/02 - Use Case - CICD/Push Change to Source Control.json [hunk 2]
```diff
@@ -73,7 +73,7 @@
                 "input": {
                     "schema": {
                         "title": "Confirmation",
-                        "description": "Do you want to Push the Change Request \"[{{vars.issue_no}}](https:\/\/github.com\/{{vars.org_name}}\/{{vars.repo_name}}\/issues\/{{vars.issue_no}})\" to the branch \"{{vars.new_branch}}\" in \"{{vars.repo_name}}\" repository?",
+                        "description": "Do you want to Push the Change Request \"[{{vars.issue_no}}](https:\/\/github.com\/{{vars.cicd_config.organizationName}}\/{{vars.repo_name}}\/issues\/{{vars.issue_no}})\" to the branch \"{{vars.new_branch}}\" in \"{{vars.repo_name}}\" repository?",
```
Comment (cs-suyog-jinturkar) [inline playbooks/02 - Use Case - CICD/Push Change to Source Control.json:76]: vars.cicd_config.organizationName --> can be org_name to follow consistent coding std

**Review by cs-suyog-jinturkar — COMMENTED** [review]
Added few minor review comments

File: playbooks/02 - Use Case - CICD/Push Change to Source Control.json (hunk 2 above)
Comment (cs-sujeet-ahirrao) [reply playbooks/02 - Use Case - CICD/Push Change to Source Control.json:76]: Only used once in playbook thats why not declared

File: playbooks/02 - Use Case - CICD/Pull Latest Content from GitHub.json (hunk 1 above)
Comment (cs-sujeet-ahirrao) [reply playbooks/02 - Use Case - CICD/Pull Latest Content from GitHub.json:816]: Step names are correct in playbook, but i have created the one step by copying the already exist thats why the route have that name even though the step name is changed

Comment (cs-suyog-jinturkar) [reply playbooks/02 - Use Case - CICD/Pull Latest Content from GitHub.json:816]: As discussed, please file a mantis for this.

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**